src/v2/Components/ArtworkFilter/ArtworkFilters/FilterExpandable.tsx

```diff
@@ -1,11 +1,67 @@
-import { Expandable, ExpandableProps, useThemeConfig } from "@artsy/palette"
-import React from "react"
+import {
+  Box,
+  Expandable,
+  ExpandableProps,
+  useThemeConfig,
+} from "@artsy/palette"
+import React, { useEffect, useRef, useState } from "react"
+import { Media } from "v2/Utils/Responsive"
+import { getElementParams } from "./helpers"
+import { useScrollRefContext } from "./useScrollContext"
+
+const FilterExpandableWithScrollIntoView: React.FC<
+  ExpandableProps & { tokens: { mb: number } }
+> = ({ tokens, ...props }) => {
+  const ctx = useScrollRefContext()
+  const scrollRef = ctx?.scrollRef
+
+  const filterRef = useRef<HTMLDivElement | null>(null)
+  const anchorRef = useRef<HTMLDivElement | null>(null)
+
+  const [open, setOpen] = useState(!!props.expanded)
+
+  useEffect(() => {
+    if (open && scrollRef?.current) {
+      const { height: containerHeight, top: containerTop } = getElementParams(
+        scrollRef.current
+      )
+      const { height: filterHeight, top: filterTop } = getElementParams(
+        filterRef.current
+      )
+      const visiblePartHeight = containerHeight - filterTop + containerTop
+
+      if (visiblePartHeight < filterHeight) {
+        anchorRef.current?.scrollIntoView({ block: "end" })
+      }
+    }
+  }, [open, filterRef])
+
+  const onClick = () => {
+    setOpen(open => !open)
+  }
+
+  return (
+    <Box ref={filterRef as any}>
+      <Expandable mb={tokens.mb} onClick={onClick} {...props} />
+      <div ref={anchorRef}></div>
+    </Box>
+  )
+}
 
 export const FilterExpandable: React.FC<ExpandableProps> = props => {
   const tokens = useThemeConfig({
     v2: { mb: 1 },
     v3: { mb: 6 },
   })
 
-  return <Expandable mb={tokens.mb} {...props} />
+  return (
+    <>
+      <Media at="xs">
+        <FilterExpandableWithScrollIntoView tokens={tokens} {...props} />
+      </Media>
+      <Media greaterThan="xs">
+        <Expandable mb={tokens.mb} {...props} />
+      </Media>
+    </>
+  )
```

Comment:
I'd consider just using `FilterExpandableWithScrollIntoView` and have it accepting a prop to enable/disable scrolling. The reason being that: there's a lot of DOM in these filters, so if possible we should avoid duplicating them. In this case we don't need to worry about the server-side rendering because any click interaction is going to happen on the client-side so we can just detect if it's mobile or not there.